core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/definition/component/ComponentJsonComponent.java

```diff
@@ -94,15 +94,14 @@ public void serialize(Component component, JsonGenerator jsonGenerator, Serializ
 
             jsonGenerator.writeObjectFieldStart(MetadataJsonComponent.Field.DATA_KEY.value());
 
-            jsonGenerator.writeStringField(Field.ID_KEY.value(), component.getMetadataKey().getId());
+            // jsonGenerator.writeStringField(Field.ID_KEY.value(), component.getMetadataKey().getId());
```

Comment:
Why is this removed?

Reply:
I compared with the json file generated from the excel . It did'nt contain the id key. And I think I got an error when I load the component to the repository. I will try again with the ID today

Reply:
what was the outcome?

Reply:
I did a mistake on first merge, I removed the comment. It add an ID to the JSON file but it is still consumed by the repository

Reply:
how do you mean, it is consumed by the repository?

Reply:
I mean, that the component is successfully saved even it has the id key

Reply:
ok